.github/ISSUE_TEMPLATE/build_error.md

```diff
@@ -1,89 +1,43 @@
 ---
-name: "\U0001F4A5 Build error" 
-about: Some package in Spack didn't build correctly  
+name: "\U0001F4A5 Build error"
+about: Some package in Spack didn't build correctly
+title: "Installation issue: "
 labels: "build-error"
 ---
 
-<!--*Thanks for taking the time to report this build failure. To proceed with the
-report please:*
+<!-- Thanks for taking the time to report this build failure. To proceed with the report please:
+
 1. Title the issue "Installation issue: <name-of-the-package>".
 2. Provide the information required below.
 
-We encourage you to try, as much as possible, to reduce your problem to the minimal example that still reproduces the issue. That would help us a lot in fixing it quickly and effectively!
--->
-
-
-### Spack version
-<!-- Add the output to the command below -->
-```console
-$ spack --version
-
-```
```

Comment:
I would keep something like this, as it permits to have an idea if the recipe in `develop` is the same as the one used in the issue.

Reply:
That’s the point of `spack debug report`

Reply:
Ah, I missed it was in the section below!

Reply:
P.S. If there's anything you want added to `spack debug report` just let me know.